robot_skills/test/test_worldmodel.py

```diff
@@ -0,0 +1,72 @@
+#! /usr/bin/env python
+
+import sys
+import rospy
+
+import robot_skills.util.kdl_conversions as kdl_conversions
+import PyKDL as kdl
+
+from robot_skills.util.robot_constructor import robot_constructor
+
+if len(sys.argv) < 2:
+    print "Please specify a robot name"
+    sys.exit()
+
+robot_name = sys.argv[1]
+
+rospy.init_node("worldmodel_test")
+
+robot = robot_constructor(robot_name)
+
+failed_actions = []
+
+#####################
+# Test Get Entities #
+#####################
+
+all_entities = robot.ed.get_entities()
+robot.speech.speak("There are {count} entities in my world".format(count=len(all_entities)))
+
+robot_loc = robot.base.get_location()  # This is just Frame, TODO make this return a FrameStamped
+here_vs = kdl_conversions.FrameStamped(robot_loc, "/map").extractVectorStamped()
```

Comment:
Will this change with your other merge request? If so, we'd better do the other one first and then merge this (otherwise we'll forget it ;))